omisego-sdk/src/main/java/co/omisego/omisego/model/transaction/consumption/TransactionConsumption.kt

```diff
@@ -146,3 +148,21 @@ data class TransactionConsumption(
         return other is TransactionConsumption && other.id == id
     }
 }
+
+/**
+ * An extension function that used the id from `TransactionConsumption` object to approve the transaction
+ *
+ * @param omgAPIClient the [OMGAPIClient] object in your application to used for approve the transaction
```

Comment:
to used for approve the transaction -> to be used to approve the transaction

Reply:
Fixed